commit 4fd68ae1a558043a2cc4ea2faf7235e71c3241aa
Author: Julia Lawall <julia@diku.dk>
Date:   Tue Oct 26 12:25:33 2010 +0200

    drivers/staging/brcm80211/brcmfmac/dhd_linux.c: delete double assignment
    
    Delete successive assignments to the same location.  dhd_ops_virt contains
    a subset of the definitions of dhd_ops_pri.
    
    A simplified version of the semantic match that finds this problem is as
    follows: (http://coccinelle.lip6.fr/)
    
    // <smpl>
    @@
    expression i;
    @@
    
    *i = ...;
     i = ...;
    // </smpl>
    
    Signed-off-by: Julia Lawall <julia@diku.dk>
    Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

--- a/drivers/staging/brcm80211/brcmfmac/dhd_linux.c
+++ b/drivers/staging/brcm80211/brcmfmac/dhd_linux.c
@@ -2222,8 +2222,6 @@ int dhd_net_attach(dhd_pub_t *dhdp, int
 	ASSERT(net);
 
 	ASSERT(!net->netdev_ops);
-	net->netdev_ops = &dhd_ops_virt;
-
 	net->netdev_ops = &dhd_ops_pri;
 
 	/*
